core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -365,6 +365,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
 
     def doControlledShutdown(retries: Int): Boolean = {
       val metadataUpdater = new ManualMetadataUpdater()
+      val logContext = new LogContext("[KafkaServer clientId=" + config.brokerId.toString + "] ")
```

Comment:
we should use  "borkerId=" here

Reply:
I thought since NetworkClient uses it as clientId, it made sense to use clientId, but you are right it could be misleading
99a77c263